assets/src/scss/components/_character-counter.scss

```diff
@@ -0,0 +1,25 @@
+.counted-field-wrapper {
+  position: relative;
+  margin-bottom: 16px;
+
+  textarea {
+    // Needed to ensure same spacing as input[type=text].
+    display: block;
+  }
+}
+
+.character-counter {
+  position: absolute;
+  bottom: -1.7em;
```

Comment:
I understand this is not part of your implementation, but what if we change to rem instead? Maybe in a follow up ticket?

Reply:
I would do this in a follow-up PR